server/api/member_test.go

```diff
@@ -40,13 +42,37 @@ type testMemberAPISuite struct {
 }
 
 func (s *testMemberAPISuite) SetUpSuite(c *C) {
-	s.hc = newHTTPClient()
+	s.hc = newUnixHTTPClient()
+
+}
+
+func unixDial(_, addr string) (net.Conn, error) {
+	return net.Dial("unix", addr)
 }
 
-func newHTTPClient() *http.Client {
-	return &http.Client{
-		Timeout: 10 * time.Second,
+func newUnixHTTPClient() *http.Client {
+	tr := &http.Transport{
+		Dial: unixDial,
 	}
+	client := &http.Client{
+		Timeout:   10 * time.Second,
+		Transport: tr,
+	}
+
+	return client
+}
+
+func mustUnixAddrToHTTPAddr(c *C, addr string) string {
+	ua, err := url.Parse(addr)
```

Comment:
so why not passing `http://localhost:1.1.sock` directly if no error.
If test, we can ensure all urls are valid.